nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-frontend/src/main/nifi/src/app/pages/cluster/state/cluster-listing/cluster-listing.effects.ts

```diff
@@ -69,6 +63,22 @@ export class ClusterListingEffects {
         )
     );
 
+    loadClusterListingSuccess$ = createEffect(() =>
+        this.actions$.pipe(
+            ofType(ClusterListingActions.loadClusterListingSuccess),
+            concatLatestFrom(() => this.store.select(selectCurrentUser)),
```

Comment:
`concatLatestFrom` is imported from the wrong package. I also noticed that `systemDiagnosticsService` is injected in the constructor here but is not used anymore.